package.json

```diff
@@ -10,13 +10,14 @@
     "test": "jest && standard"
   },
   "dependencies": {
+    "joi": "^13.1.2",
     "probot": "^5.0.0",
     "probot-scheduler": "^1.0.2",
     "standard": "^10.0.3"
   },
   "engines": {
-    "node": "^7.7",
-    "npm": "^4.0"
+    "node": "^8.9",
+    "npm": "^5.6"
```

Comment:
just a comment, since npm 3, I never run into a situation where setting the npm in engines would help. And for the node engine, I would recommend to just set it to "8". As Deployment targets update their versions you can run in a situation where there no longer is a version `8.9.x` supported and then it will fail, although there might be `8.10.1`. Just setting it to "8" gives you the latest Node 8 version they got